draft-ietf-ice-rfc5245bis.xml

```diff
@@ -3220,23 +2934,6 @@ in <xref target="RFC5766"/>.
 
 </section>
 
-<section title="Procedures for All Implementations">
-
-<t> ICE has interactions with jitter buffer adaptation mechanisms. An
-RTP stream can begin using one candidate, and switch to another one,
-though this happens rarely with ICE. The newer candidate may result in
-RTP packets taking a different path through the network -- one with
-different delay characteristics. As discussed below, agents are
-encouraged to re-adjust jitter buffers when there are changes in
-source or destination address of media packets. Furthermore, many
-audio codecs use the marker bit to signal the beginning of a
-talkspurt, for the purposes of jitter buffer adaptation. For such
-codecs, it is RECOMMENDED that the sender set the marker bit <xref
-target="RFC3550"/> when an agent switches transmission of media from
-one candidate pair to another.  </t>
-
```

Comment:
I don't think such RTP-specific things should be a part of the ICE spec.  The ICE spec should be about how to make ICE interop, not about what to do with RTP when using ICE.   But, sure let the WG decide at IETF 99.  

And, yes, this is another non-editorial.  Sorry about that.  Do you want me to make a separate PR/issue/thread?